src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -50,7 +54,7 @@ public class ResourceManagerStringLocalizer : IStringLocalizer
         /// </summary>
         public ResourceManagerStringLocalizer(
             ResourceManager resourceManager,
-            AssemblyWrapper resourceAssemblyWrapper,
+            IResourceStreamManager resourceStreamManager,
```

Comment:
Unless I misunderstood him @rynowak was of the opinion that any ctor that had an internal type as a parameter (like `AssemblyWrapper`) was to be considered internal and we didn't have to worry too much about breaking it.